pkg/kube/mock_client.go

```diff
@@ -80,6 +80,9 @@ type MockClient struct {
 	IstiodVersion     string
 }
 
+func (c MockClient) SetPortManager(manager PortManager) {
```

Comment:
should this panic, so it's obvious from the tests if we call it? Or do we expect it to be called?